### PR TITLE
CVE Records for iNotes

### PR DESCRIPTION
**Pull request by monicamurray**

### CODE REVIEW

File: 2022/27xxx/CVE-2022-27547.json [hunk 1]
```diff
@@ -1,18 +1,84 @@
 {
```
Comment (cve-team) [inline 2022/27xxx/CVE-2022-27547.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/27xxx/CVE-2022-27547.json:1]: ("Expecting ',' delimiter: line 85 column 1 (char 2706)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/27xxx/CVE-2022-27547.json [hunk 2]
```diff
@@ -1,18 +1,84 @@
...
+        "reference_data": [
+            {
+                "refsource": "CONFIRM",
+                "url": "https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0100212"
+            }
+        ]
+    },
+    "source": {
+        "discovery": "UNKNOWN"
     }
```
Comment (anthnysingleton) [inline 2022/27xxx/CVE-2022-27547.json:84]: The error here is that you are missing the Closing '}' for this json record.

**Pull request merged**